TCAT/Supporting Files/Info.plist

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>Fabric</key>
+    <dict>
+        <key>APIKey</key>
+        <string>50d77fbd594f6229f8bb9a3669ce18dce7dac38e</string>
```

Comment:
Did you add this? We might want to throw this in our config.json so it doesn't go on the repo